src/lib/webpack/webpack-client-config.js

```diff
@@ -199,7 +199,12 @@ const production = config => addPlugins([
 			/push-manifest\.json$/,
 			/.DS_Store/
 		]
-	})
+	}),
+	new webpack.DefinePlugin({
```

Comment:
I had posted in chat about this but I'll echo here for visibility - does the object form here replace as optimally as this?

```js
new webpack.DefinePlugin({
  'process.env.ADD_SW': config.serviceWorker
});
```

Reply:
I believe `yargs` will have camelCased the flag name at this point.

@prateekbh Have you checked/ran this? This was my thought as well.